### PR TITLE
Change the 'show image' styling

### PR DESCRIPTION
**Pull request by GretaD**
Fixes https://github.com/nextcloud/mail/issues/6122

this is how i tried to make it work with the state of actions we have now.
But i agree with Marco's comment here: https://github.com/nextcloud/mail/issues/6122#issuecomment-1202480324

before
Screenshot from 2022-08-02 15-35-35
Screenshot from 2022-08-02 15-35-22

after
hvcfgh

png

~~- [ ] Requires https://github.com/nextcloud/nextcloud-vue/pull/2911~~

### CODE REVIEW

**Comment by GretaD** [comment]
@jancborchardt and @nimishavijay please have a look at this solution and if you're ok with it

**Review by jancborchardt — CHANGES_REQUESTED** [review]
- [x] "Show image" → "Show images"
- [x] The eye icon is not necessary, as per mockup
- [x] In the menu, the icons are all the same, so either we should go with no icons or:
    - Temporarily: https://fonts.google.com/icons?selected=Material%20Icons%3Aimage%3A
    - Show from this email: https://fonts.google.com/icons?selected=Material%20Icons%3Amail%3A
    - Show from this domain: https://fonts.google.com/icons?selected=Material%20Icons%3Adomain%3A
    
Regarding the comment from @marcoambrosini, either is fine with me. Indeed using the tertiary button component would be more accurate since it’s not a link which goes somewhere.

**Comment by GretaD** [comment]
> * [ ]  "Show image" → "Show images"
> * [ ]  The eye icon is not necessary, as per mockup
> * [ ]  In the menu, the icons are all the same, so either we should go with no icons or:
>   
>   * Temporarily: https://fonts.google.com/icons?selected=Material%20Icons%3Aimage%3A
>   * Show from this email: https://fonts.google.com/icons?selected=Material%20Icons%3Amail%3A
>   * Show from this domain: https://fonts.google.com/icons?selected=Material%20Icons%3Adomain%3A
> 
> Regarding the comment from @marcoambrosini, either is fine with me. Indeed using the tertiary button component would be more accurate since it’s not a link which goes somewhere.

all the point can be delivered but the icon one. The action must have an icon, sorry should have commented that before. If we wait for the vue changes, this will be delivered a bit late, if we move on with my solution, i will look like this(and your comments, but with an icon)

**Comment by jancborchardt** [comment]
@GretaD cool, and fine for now with the icon too, but then also use the "image" filetype icon instead of "eye".

**Comment by marcoambrosini** [comment]
We can definitely add the option not to have an icon and have the menus triggered by tex-tonly buttons. I think it totally makes sense.

**Comment by marcoambrosini** [comment]
...and that will automatically be the case when https://github.com/nextcloud/nextcloud-vue/issues/2768 happens

**Comment by GretaD** [comment]
> We can definitely add the option not to have an icon and have the menus triggered by tex-tonly buttons. I think it totally makes sense.

i know that, but when it is going to be released? :)

**Comment by marcoambrosini** [comment]
pr is there https://github.com/nextcloud/nextcloud-vue/pull/2911#issuecomment-1207891395
Let's wait for more reviewers since it's a big change. Also you can review @GretaD :) 

Once that's we can do a minor release

**Comment by ChristophWurst** [comment]
How is this unblocked when it depends on https://github.com/nextcloud/nextcloud-vue/pull/2911 and this app still uses v5?

**Comment by GretaD** [comment]
> How is this unblocked when it depends on [nextcloud/nextcloud-vue#2911](https://github.com/nextcloud/nextcloud-vue/pull/2911) and this app still uses v5?

ah sorry didnt see you added the component on desc. I am not using that component, that was not related to actions without icons, what Marco meant(double checked with him) is that after that is in, we can edit the actions on vue again and add a slot with no icons. 

What i did was hiding the icon with display none. 
I dont have time to spend on the component now, so we can either do this or wait till we have time.

File: src/components/MessageHTMLBody.vue [hunk 1]
```diff
@@ -3,19 +3,26 @@
 		<MdnRequest :message="message" />
 		<div v-if="hasBlockedContent" id="mail-message-has-blocked-content" style="color: #000000">
 			{{ t('mail', 'The images have been blocked to protect your privacy.') }}
-			<Actions default-icon="icon-toggle" :menu-title="t('mail', 'Show')">
-				<ActionButton icon="icon-toggle"
+			<Actions type="tertiary" :menu-title="t('mail', 'Show image')">
```
Comment (jancborchardt) [inline src/components/MessageHTMLBody.vue:6]:
```suggestion
			<Actions type="tertiary" :menu-title="t('mail', 'Show images')">
```

**Review by jancborchardt — CHANGES_REQUESTED** [review]
Except changing it to plural it looks good to me :)

**Comment by GretaD** [comment]
not ready to merge

**Comment by GretaD** [comment]
> not ready to merge

ready now, please reviewers test it

**Review by ChristophWurst — APPROVED** [review]
Tested, works

**Pull request merged**